### DISCUSSION TITLE
Bump @xstate/immer dependency to > v9.0.6

### DISCUSSION THREAD

**Discussion by rflow** [statelyai/xstate#2749] [General]
[Recent security advice](https://www.cvedetails.com/cve/CVE-2021-23436/) state versions of immer prior to v9.0.6  contain a vulnerability that allows security restrictions to be bypassed.

Wondering if it might be possible to update [this package](https://github.com/statelyai/xstate/tree/main/packages/xstate-immer) to use a more recent version?

**Comment by rflow** [comment]
FWIW - I've tried updating the dependency locally and running the package tests. Everything appears to work fine. Happy to submit a PR if that helps?

**Reply by davidkpiano** [reply]
Yes please!

**Reply by rflow** [reply]
👍 https://github.com/statelyai/xstate/pull/2750
